harness/determined/_trial_controller.py

```diff
@@ -37,15 +37,16 @@ class TrialController(metaclass=abc.ABCMeta):
     def __init__(
         self,
         context: Any,
-        env: det.EnvContext,
+        env: Optional[det.EnvContext] = None,
```

Comment:
I vote: don't even have `PyTorchTrialController` inherit from `TrialController` anymore.  It's now it's own thing, and the root `TrialController` belongs to legacy crap, that will be `rm -rf`'d.

Reply:
done